test/TokenTestUtils.js

```diff
@@ -166,128 +171,43 @@ function checkBurnEvents(burning, amount, burner) {
 
 }
 
+var fiatTokenEmptyState = {
+    "name": name,
+    "symbol": symbol,
+    "currency": currency,
+    "decimals": new BigNumber(decimals),
+    "masterMinter": Accounts.masterMinterAccount,
+    "pauser": Accounts.pauserAccount,
+    "blacklister" : Accounts.blacklisterAccount,
+    "tokenOwner": Accounts.tokenOwnerAccount,
+    "proxiedTokenAddress": bigZero,
+    "initializedV1": trueInStorageFormat,
+    "proxyOwner": Accounts.proxyOwnerAccount,
+    "balances": setAccountDefault(Accounts, bigZero),
+    "allowance": recursiveSetAccountDefault(Accounts, bigZero),
+    "totalSupply": bigZero,
+    "isAccountBlacklisted": setAccountDefault(Accounts, false),
+    "isAccountMinter": setAccountDefault(Accounts, false),
+    "minterAllowance": setAccountDefault(Accounts, bigZero),
+    "paused": false,
+};
+
+
+function recursiveSetAccountDefault(accounts, value) {
```

Comment:
We might want to move this to `AccountUtils`

Reply:
fixed.